aws/rust-runtime/aws-sigv4/src/http_request/sign.rs

```diff
@@ -93,6 +95,35 @@ pub enum SignableBody<'a> {
     StreamingUnsignedPayloadTrailer,
 }
 
+/// Formats the value using the given formatter. To print the body data, set `LOG_SIGNABLE_BODY=true`.
+impl<'a> Debug for SignableBody<'a> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let should_log_signable_body = std::env::var(LOG_SIGNABLE_BODY)
+            .map(|v| v.eq_ignore_ascii_case("true"))
+            .unwrap_or_default();
+        match self {
+            Self::Bytes(arg0) => {
+                if should_log_signable_body {
+                    f.debug_tuple("Bytes").field(arg0).finish()
+                } else {
+                    f.debug_tuple("Bytes").finish()
```

Comment:
Should we log something like `"body redacted"` or a pointer to the `LOG_SIGNABLE_BODY` env var in place of the actual value?

Reply:
Interestingly, that's what a function [log_sensitive_body](https://github.com/smithy-lang/smithy-rs/blob/df77d5ff33f143abaea3f8bf5a9cdeec05353ece/rust-runtime/aws-smithy-runtime/src/client/orchestrator/http.rs#L50-L51) does in the trace log message. Probably a pointer to `LOG_SIGNABLE_BODY` would be preferred in this case. Will make that adjustment.